<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1064</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1041</width>
      <height>431</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QFrame" name="frameLeft">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>图像属性</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="6" column="1">
          <widget class="QLineEdit" name="editImg_DPM"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>图像深度</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>图像格式</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>DPI分辨率</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="editImg_Format"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>图像高度</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>图像宽度</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>位平面数</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QCheckBox" name="chkBox_Alpha">
           <property name="text">
            <string>具有Alpha通道</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="editImg_Height"/>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="editImg_Width"/>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="editImg_Depth"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>图像大小</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="editImg_BitPlane"/>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="editImg_SizeByte"/>
         </item>
         <item row="7" column="0">
          <widget class="QCheckBox" name="chkBox_GrayScale">
           <property name="text">
            <string>灰度图</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>图像格式转换</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QComboBox" name="comboFormat">
           <item>
            <property name="text">
             <string>16-bit RGB565 format</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>24-bit RGB888 format</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>32-bit RGB format</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>8-bit grayscale format</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>16-bit grayscale format</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>8-bit indexes into a colormap</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnFormatConvert">
           <property name="text">
            <string>图像格式转换</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/images/824.bmp</normaloff>:/images/824.bmp</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>RGB565数据</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QRadioButton" name="radioLSB">
             <property name="text">
              <string>低字节在前</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioMSB">
             <property name="text">
              <string>高字节在前</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="btnGetRGB565">
           <property name="text">
            <string>生成RGB565数据</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/images/116.bmp</normaloff>:/images/116.bmp</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSaveDataFile">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>保存数据文件</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/images/save.bmp</normaloff>:/images/save.bmp</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/images/824.bmp</normaloff>:/images/824.bmp</iconset>
      </attribute>
      <attribute name="title">
       <string>图像</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QWidget" name="scrollAreaWidgetContents" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="labPic">
            <property name="text">
             <string>图片</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/images/116.bmp</normaloff>:/images/116.bmp</iconset>
      </attribute>
      <attribute name="title">
       <string>RGB565</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QPlainTextEdit" name="plainTextEdit"/>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actFile_Open"/>
   <addaction name="actFile_Reload"/>
   <addaction name="actFile_Save"/>
   <addaction name="actFile_SaveAs"/>
   <addaction name="actFile_Preview"/>
   <addaction name="actFile_Print"/>
   <addaction name="separator"/>
   <addaction name="actImg_ZoomIn"/>
   <addaction name="actImg_ZoomOut"/>
   <addaction name="actImg_RotateLeft"/>
   <addaction name="actImg_RotateRight"/>
   <addaction name="actImg_FlipUD"/>
   <addaction name="actImg_FilpLR"/>
   <addaction name="separator"/>
   <addaction name="actFile_Quit"/>
  </widget>
  <action name="actFile_Open">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/001.GIF</normaloff>:/images/001.GIF</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actFile_Quit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/132.bmp</normaloff>:/images/132.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actFile_Save">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/save.bmp</normaloff>:/images/save.bmp</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actImg_RotateLeft">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/rotateleft.png</normaloff>:/images/rotateleft.png</iconset>
   </property>
   <property name="text">
    <string>左旋</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actImg_RotateRight">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/rotateright.png</normaloff>:/images/rotateright.png</iconset>
   </property>
   <property name="text">
    <string>右旋</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actFile_SaveAs">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/save-as24.png</normaloff>:/images/save-as24.png</iconset>
   </property>
   <property name="text">
    <string>另存</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actFile_Preview">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/preview1.bmp</normaloff>:/images/preview1.bmp</iconset>
   </property>
   <property name="text">
    <string>打印预览</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actFile_Print">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/fileprint.png</normaloff>:/images/fileprint.png</iconset>
   </property>
   <property name="text">
    <string>打印</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actImg_ZoomIn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/zoomin.png</normaloff>:/images/zoomin.png</iconset>
   </property>
   <property name="text">
    <string>放大</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actImg_ZoomOut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/zoomout.png</normaloff>:/images/zoomout.png</iconset>
   </property>
   <property name="text">
    <string>缩小</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actImg_FilpLR">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/rotate-anticlockwise.png</normaloff>:/images/rotate-anticlockwise.png</iconset>
   </property>
   <property name="text">
    <string>左右翻转</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actImg_FlipUD">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/shape-rotate-clockwise-icon.png</normaloff>:/images/shape-rotate-clockwise-icon.png</iconset>
   </property>
   <property name="text">
    <string>上下翻转</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actFile_Reload">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/126.bmp</normaloff>:/images/126.bmp</iconset>
   </property>
   <property name="text">
    <string>重新载入</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actFile_Quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>533</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
